paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/ui/wallet/SavedPaymentMethodsViewModel.kt

```diff
@@ -28,51 +30,75 @@ class SavedPaymentMethodsViewModel(
     )
     val state: StateFlow<SavedPaymentMethodsViewState> = _state
 
-    init {
-        initialize()
-    }
+    internal val customerAdapter: CustomerAdapter = CustomerAdapter.create(
+        context = getApplication(),
+        customerEphemeralKeyProvider = {
+            fetchCustomerEphemeralKey().fold(
+                success = {
+                    kotlin.Result.success(
+                        CustomerEphemeralKey.create(
+                            customerId = it.customerId,
+                            ephemeralKey = it.customerEphemeralKeySecret
+                        )
+                    )
+                },
+                failure = {
+                    kotlin.Result.failure(it.exception)
+                }
+            )
+        },
+        setupIntentClientSecretProvider = null,
```

Comment:
Do/should these have defaults?

Reply:
I am not sure what the best approach is here. We want merchants to use the `setupIntentClientSecretProvider` as the default. Not using this is the more advanced flow that we want merchants to avoid.

Reply:
Given that info, I think it makes sense to leave as is. Are you planning to actually use `setupIntentClientSecretProvider` in a follow up?

Reply:
Yep!